### ISSUE TITLE
YOLO loss function

### ISSUE THREAD

**Issue by kavindujp** [AlexeyAB/darknet#8479]
I want to implement and train yolov3 using tensorflow. I want to know about loss function used in YOLO. But I could not find information regarding that. Anyone know what is the Loss function used in original YOLO and what is the optimization algorithm used in YOLO?

**Comment by kavindujp** [comment]
> I want to implement and train yolov3 using tensorflow. I want to know about loss function used in YOLO. But I could not find information regarding that. Anyone know what is the Loss function used in original YOLO and what is the optimization algorithm used in YOLO?
